### ISSUE TITLE
Error 0x800002C3 connector to database

### ISSUE THREAD

**Issue by iTeo85** [mhertzfeld/CrystalReportsNinja#36]
Hi,
i use CrystalReportsNinja v1.8.0.0.x64 on my pc, and i have some problems to use the reports with subreports; if i don't link the subreport with main reports CrystalReportsNinja reads and exports correctly the pdf, but when i set the parameters for link the subreport with main report CrystalReportsNinja return me Error 0x800002C3.
Can you help me to understand where i mistake ?

P.S
reports work correctly in Crystal Report XI

attached are the logs of the two cases I have described:
[ninja-20210113163409_Success.log](https://github.com/mhertzfeld/CrystalReportsNinja/files/5809960/ninja-20210113163409_Success.log)
[ninja-20210113164310_Error.log](https://github.com/mhertzfeld/CrystalReportsNinja/files/5809961/ninja-20210113164310_Error.log)

**Comment by mhertzfeld** [comment]
Issues with subreports come up a lot.  The issues are difficult to troubleshoot since for me to troubleshoot the report needs to be recreated and able to replicate the problem using a database I have access to (adventure works) so that I can debug.

Take a look at some of the closed issues and see if anything sounds familiar.

If you can build a report using adventure works that has the same problem will try to debug the problem.  There are a few sample reports you can use for this in the testing directory.

https://github.com/mhertzfeld/CrystalReportsNinja/tree/master/Testing/AdventureWorksLt

**Comment by c-josewski** [comment]
From your error log it says you have (8) detected parameters and (6) items that say that they are marked as 'Skipped'.
From your success log it says you have (6) detected parameters and (4) items marked as 'Skipped'.

I think I had a similar error, though I am not sure on the particular error code it gave me. Anyway the problem for me was that I was not passing the variables of the sub-report in my print command. I ended up fixing this within the main report because the sub-report was not linked how I really intended. Check to make sure the sub-report is handling the prompt fields correctly.

I ended up creating an extremely basic report with a sub-report and getting that to print to finally figure out what I did wrong. I hope this helps find the problem.
